Makefile

```diff
@@ -62,7 +74,15 @@ generate: $(BIN)/buf $(BIN)/license-header ## Regenerate code and licenses
 
 .PHONY: upgrade
 upgrade: ## Upgrade dependencies
-	go get -u -t ./... && go mod tidy -v
+	@UPDATE_PKGS=$$($(GO) list -u -f '{{if and .Update (not (or .Main .Indirect .Replace))}}{{.Path}}@{{.Update.Version}}{{end}}' -m all); \
```

Comment:
Upgrades only direct dependencies and also updates `buf.gen.yaml` to keep that version in sync with the protobuf module version.

Reply:
Thanks!